packages/mobile/src/screens/DetailsScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  Button,
+  StatusBar,
+} from 'react-native';
+
+const DetailsScreen = ({navigation}) => {
+    return (
+      <View style={{flex: 1, alignItems: 'center', justifyContent: 'center'}}>
```

Comment:
jak style nie zależą od wyliczeń/ propsów to nie stylujmy inline. Style w widoku zaciemniają kod